tools/packaging/packaging.psm1

```diff
@@ -30,226 +30,261 @@ function Start-PSPackage {
 
         [Switch] $Force,
 
-        [Switch] $IncludeSymbols,
-
         [Switch] $SkipReleaseChecks
     )
 
     # The package type 'deb-arm' is current disabled for '-Type' parameter because 'New-UnixPackage' doesn't support
     # creating package for 'deb-arm'. It should be added back to the ValidateSet of '-Type' once the implementation
     # of creating 'deb-arm' package is done.
 
-    # Runtime and Configuration settings required by the package
-    ($Runtime, $Configuration) = if ($WindowsRuntime) {
-        $WindowsRuntime, "Release"
-    } elseif ($Type -eq "deb-arm") {
-        New-PSOptions -Configuration "Release" -Runtime "Linux-ARM" -WarningAction SilentlyContinue | ForEach-Object { $_.Runtime, $_.Configuration }
-    } else {
-        New-PSOptions -Configuration "Release" -WarningAction SilentlyContinue | ForEach-Object { $_.Runtime, $_.Configuration }
-    }
+    DynamicParam {
+        if ($Type -eq "zip") {
+            # Add a dynamic parameter '-IncludeSymbols' when the specified package type is 'zip'.
+            # The '-IncludeSymbols' parameter can be used to indicate that the package should only contain powershell binaries and symbols.
+            $ParameterAttr = New-Object "System.Management.Automation.ParameterAttribute"
+            $Attributes = New-Object "System.Collections.ObjectModel.Collection``1[System.Attribute]"
+            $Attributes.Add($ParameterAttr) > $null
 
-    if($Environment.IsWindows) {
-        # Runtime will always be win7-x64 or win7-x86 on Windows.
-        # Build the name suffix for universal win-plat packages.
-        $NameSuffix = $Runtime -replace 'win\d+', 'win'
+            $Parameter = New-Object "System.Management.Automation.RuntimeDefinedParameter" -ArgumentList ("IncludeSymbols", [switch], $Attributes)
+            $Dict = New-Object "System.Management.Automation.RuntimeDefinedParameterDictionary"
+            $Dict.Add("IncludeSymbols", $Parameter) > $null
+            return $Dict
+        }
     }
 
-    log "Packaging RID: '$Runtime'; Packaging Configuration: '$Configuration'"
+    End {
+        $IncludeSymbols = $null
+        if ($PSBoundParameters.ContainsKey('IncludeSymbols')) {
+            log 'setting IncludeSymbols'
+            $IncludeSymbols = $PSBoundParameters['IncludeSymbols']
+        }
+        log "$($IncludeSymbols.IsPresent):$IncludeSymbols"
```

Comment:
When `-IncludeSymbols` is not specified, this log will print just a colon.

Reply:
I'll remove... this was troubleshooting...